src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java

```diff
@@ -173,6 +187,15 @@ public boolean isLogoutLinkVisible() {
     }
   }
 
+  public boolean isExploreWikiHeaderVisible() {
+    try {
+      return exploreWikiHeader.isDisplayed();
```

Comment:
@ludwikkazmierczak - what would be the most appropriate place given the current architectural direction for selenium tests?